### PR TITLE
Web specific locators for Appium

### PR DESCRIPTION
**Pull request by DavertMik**
To have one test for mobile and web app.

* added `runInWeb` method to execute code only in web
* added `web` locator type similarly to `android` and `ios`

### CODE REVIEW

File: lib/helper/Appium.js [hunk 1]
```diff
@@ -276,6 +276,26 @@ class Appium extends WebdriverIO {
...
+   * ```js
+   * I.runInWeb(() => {
+   *    I.waitForElement('#data');
+   *    I.seeInCurrentUrl('/data');
+   * });
+   * ```
+   *
+   * @param {*} fn
+   */
+  runInWeb(fn) {
```
Comment (APshenkin) [inline lib/helper/Appium.js]: will this works with generators? Think, that we should change generators to awync/await, when Node 8 will be LTS

File: lib/helper/Appium.js [hunk 2]
```diff
@@ -276,6 +276,26 @@ class Appium extends WebdriverIO {
...
+    if (!this.isWeb) return;
+    recorder.session.start('Web-only actions');
+    fn();
+    recorder.add('restore from Web session', () => recorder.session.restore());
```
Comment (APshenkin) [inline lib/helper/Appium.js]: I think, we should have tests for this cool features like different locators for different platforms, execution only on specified platform. This is killer feature and we have to be sure that it's works after each update

Comment (DavertMik) [inline lib/helper/Appium.js]: Yes, there are unit tests for Android / iOS. I will add few more for Web

File: lib/helper/Appium.js (hunk 1 above)
Comment (DavertMik) [inline lib/helper/Appium.js]: Oh, generators... I will try to apply them.

**Pull request merged**